config_repo/config.sh.repo

```diff
@@ -1,139 +1,187 @@
-TIMELAPSE=true
 
-# Set the width and height parameter for timelapse generation (sizes must be divisible by 2)
+########## Timelapse
+# Set to "true" to generate a timelapse video at the end of each night.
+TIMELAPSE="true"
+
+# Set the image width and height for timelapse video (sizes must be divisible by 2).
 TIMELAPSEWIDTH=0
 TIMELAPSEHEIGHT=0
 
-# Set the bitrate of the timelapse video.  Higher numbers will produce higher quality but larger files.
+# Bitrate of the timelapse video.  Higher numbers will produce higher quality but larger files.
 TIMELAPSE_BITRATE="2000k"
 
-# Timelapse frame rate (number of frames per second)
+# Timelapse video Frames Per Second.
 FPS=25
 
-# Encoder for timelapse. May be changed to use a hardware encoder or different codec
-VCODEC=libx264
-
-# Set to true to generate a keogram at the end of the night (image summary of the night)
-KEOGRAM=true
+# Encoder for timelapse video. May be changed to use a hardware encoder or different codec.
+VCODEC="libx264"
 
-# Set to true to upload the keogram after it has been generated
-UPLOAD_KEOGRAM=false
+# FFLOG determines the amount of log information displayed while creating a timelapse video.
+# Set to "info" to see additional information if you are tuning the algorithm.
+FFLOG="warning"
 
-# Set to true to generate a startrails image of the night. Will skip brighter images to avoid over-exposure
-STARTRAILS=true
+# While creating a timelapse video, a sequence of links to the images is created.
+# Set to "1" to keep that sequence; set to null ("") to have it deleted when done.
+KEEP_SEQUENCE=""
 
-# Images with a brightness higher than the threshold will be skipped for startrails image generation
-BRIGHTNESS_THRESHOLD=0.1
+# Any additional timelapse parameters.  Run "ffmpeg -?" to see the options.
+TIMELAPSE_PARAMETERS=""
 
-# Set to true to upload the startrails after it has been generated
-UPLOAD_STARTRAILS=false
+# Set to "true" to upload timelapse video to your website at the end of each night.
+UPLOAD_VIDEO="false"
 
-# Set to true to enable automatic deletion of archived data (images + videos)
-AUTO_DELETE=true
 
-# Set this value to the number of archived nights you want to keep. Needs AUTO_DELETE=true to work
-NIGHTS_TO_KEEP=14
+########## Keogram
+# Set to "true" to generate a keogram at the end of each night.
+KEOGRAM="true"
 
-# Set to true if you want to enable dark frame subtraction on your night time images
-DARK_FRAME_SUBTRACTION=false
+# Keogram parameters.  Run "scripts/keogram -?" to see the options.
+# Keograms work best if North is at the top of the image.  If this is not the case you can
+# rotate the image; for example, if North is straight down in your image, rotate it 180 degrees.
+# Rotation is anticlockwise. 0 disables rotation.
+# Monochrome cameras have black and white text and only the first number in fontcolor is used.
+KEOGRAM_PARAMETERS="-rotate 0 -fontsize 1.0 -fontline 1 -fontcolor 255 255 255"
```

Comment:
Incorrect flag syntax for keogram. Additionally, 0 rotation is unnecessary, since that's the default.

```
Usage:	keogram -d <imagedir> -e <ext> -o <outputfile> [<other_args>]

Arguments:
-d | --directory <str> : directory from which to load images (required)
-e | --extension <str> : image extension to process (required)
-o | --output-file <str> : name of output file (required)
-r | --rotate <float> : number of degrees to rotate image, counterclockwise (0)
-s | --image-size <int>x<int> : only process images of a given size, eg. 1280x960
-h | --help : display this help message
-v | --verbose : Increase logging verbosity
-n | --no-label : Disable hour labels
-C | --font-color <str> : label font color, in HTML format (0000ff)
-L | --font-line <int> : font line thickness (3)
-N | --font-name <str> : font name (simplex)
-S | --font-size <float> : font size (2.0)
-T | --font-type <int> : font line type (1)
```